### ISSUE TITLE
php-packages.imagick doesn't build on 17.03

### ISSUE THREAD

**Issue by vaibhavsagar** [NixOS/nixpkgs#29070]
## Issue description
The `imagick` PHP extension has been broken for the past couple of `17.03` releases, but it is fixed in `master` and the current `17.09` branch. I was able to trace this back to https://github.com/NixOS/nixpkgs/commit/d33b18c57b5ae727ad45ac3fd68248d9b6014e44.

### Steps to reproduce

I used `git bisect` as follows:

```bash
[me@nixos:~/repos/nixpkgs]$ git checkout channels/nixos-17.03
[me@nixos:~/repos/nixpkgs]$ cat <<EOF > script.sh
> #!/bin/env bash
> nix-build --no-out-link -E "with (import ./default.nix {}); (pkgs.callPackage ./pkgs/top-level/php-packages.nix {}).imagick"
> EOF

[me@nixos:~/repos/nixpkgs]$ chmod +x script.sh
[me@nixos:~/repos/nixpkgs]$ git bisect start HEAD 1b57bf274ae5c76e91b2b264d8aa8bfcecb72102
[me@nixos:~/repos/nixpkgs]$ git bisect run ./script.sh
(lots of output...)
d33b18c57b5ae727ad45ac3fd68248d9b6014e44 is the first bad commit
commit d33b18c57b5ae727ad45ac3fd68248d9b6014e44
Author: Tim Steinbach <tim@nequissimus.com>
Date:   Sat Jul 29 13:40:13 2017 -0400

    ImageMagick: 6.9.9-0 -> 6.9.9-3
    
    (cherry picked from commit bde8a15019a21fff28cac8ad82a39c5f30964717)

:040000 040000 134cbb5faa98b0ac393931ccb29e36d72ffb61c6 96b111627efb9720635944ce42ffad9bc3d55a49 M      pkgs
bisect run success
```

## Technical details

* System: `17.03.1810.6c8b819c99 (Gorilla)`
* Nix version: `nix-env (Nix) 1.11.14`
* Nixpkgs version: `6c8b819c99a85276f9d3ebdefdb039235321c646`
* Sandboxing enabled: no

**Comment by bjornfor** [comment]
Fixed on release-17.03: a0e6a891ee21a6dcf3da35169794cc20b110ce05.

**Comment by vaibhavsagar** [comment]
Thanks!
